app.py

```diff
@@ -23,7 +24,10 @@ def index():
     @app.route("/notifications", methods=['POST', 'GET'])
     def calendar_notifications():
         PushNotification().send_notifications_to_subscribers()
-        return PushNotification.send_notifications(PushNotification)
+        job = PushNotification.send_notifications(PushNotification)
+        PushNotification.send_notifications.delay(job)
+        return job
```

Comment:
Celery was built on the truth, which was to to make user experience better by not allowing users, to have to wait for certain tasks to run. The code above will run forever i.e it will not return anything and hence python will just keep running it waiting for a return i.e if the browser does not declare a timeout. A Job(delay), is a function which tells celery to run that function it has been scheduled on, using that variable in fact can be quite confusing as it represents something different in this context(returning data from `PushNotification.send_notifications(PushNotification)`). In a nutshell, when you schedule a job(Delay) in Celery, you actually send back an immediate message to the user that the request succeeded or was made and please use a different variable that can at least be more descriptive in the context of which it is used